packages/password/tests/apple.password.test.js

```diff
@@ -0,0 +1,61 @@
+const { internal } = require('../lib/apple.password')
+const fc = require('fast-check')
+
+describe('Apple password implementations', () => {
+    let n = []
+    beforeAll(() => {
+        n = []
+    })
+    afterAll(() => {
+        console.log(n)
+    })
+    it('should produce positive integers from ranges _randomNumberWithUniformDistribution', () => {
+        fc.assert(
+            fc.property(fc.integer({min: 1}), data => {
+                const result = internal._randomNumberWithUniformDistribution(data)
+                return result >= 0
+            }),
+            { seed: 1691610651, path: '0', endOnFailure: true }
```

Comment:
Not sure what this stuff is. Can you please explain?

Reply:
It was left there by mistake.

A feature of `fast-check`, is that when it's generating random inputs for your methods, if it catches a bug, you need it to **stop** being random so that you can debug - `fast-check` gives this data on the command line after a failure, I just forgot to remove it :)